diff -Nur lpr-0.33/common_source/pathnames.h lpr-0.33.pld/common_source/pathnames.h
--- lpr-0.33/common_source/pathnames.h	Sun Apr 19 00:37:03 1998
+++ lpr-0.33.pld/common_source/pathnames.h	Mon Jun  7 09:30:30 1999
@@ -40,12 +40,12 @@
 #define	_PATH_DEFSPOOL		"/var/spool/lpd"
 #define	_PATH_HOSTSEQUIV	"/etc/hosts.equiv"
 #define	_PATH_HOSTSLPD		"/etc/hosts.lpd"
-#define	_PATH_MASTERLOCK	"/var/spool/lpd/lpd.lock"
+#define	_PATH_MASTERLOCK	"/var/lock/lpd.lock"
 #define	_PATH_PR		"/usr/bin/pr"
 #define	_PATH_PRINTCAP		"/etc/printcap"
 #define	_PATH_SOCKETNAME	"/dev/printer"
-#define	_PATH_VFONT		"/usr/libdata/vfont/"
-#define	_PATH_VFONTB		"/usr/libdata/vfont/B"
-#define	_PATH_VFONTI		"/usr/libdata/vfont/I"
-#define	_PATH_VFONTR		"/usr/libdata/vfont/R"
-#define	_PATH_VFONTS		"/usr/libdata/vfont/S"
+#define	_PATH_VFONT		"/usr/share/fonts/vfont/"
+#define	_PATH_VFONTB		"/usr/share/fonts/vfont/B"
+#define	_PATH_VFONTI		"/usr/share/fonts/vfont/I"
+#define	_PATH_VFONTR		"/usr/share/fonts/vfont/R"
+#define	_PATH_VFONTS		"/usr/share/fonts/vfont/S"
--- lpr-0.72/lpq/Makefile.orig	Thu May 17 23:53:46 2001
+++ lpr-0.72/lpq/Makefile	Mon Mar 24 17:38:49 2003
@@ -1,6 +1,6 @@
 
-CFLAGS	= -Wall $(RPM_OPT_FLAGS) -I../common_source -I/usr/include/bsd
-LDFLAGS	= 
+CFLAGS	= -Wall $(OPT_FLAGS) -I../common_source -I/usr/include/bsd
+LDFLAGS	= -s
 LDLIBS = -lcommon -L ../common_source # -lbsd
 
 all:	lpq
--- lpr-0.72/lpr/Makefile.orig	Thu May 17 23:53:46 2001
+++ lpr-0.72/lpr/Makefile	Mon Mar 24 17:39:08 2003
@@ -1,6 +1,6 @@
 
-CFLAGS	= -Wall $(RPM_OPT_FLAGS) -I../common_source -I/usr/include/bsd
-LDFLAGS	= 
+CFLAGS	= -Wall $(OPT_FLAGS) -I../common_source -I/usr/include/bsd
+LDFLAGS	= -s
 LDLIBS = -lcommon -L ../common_source # -lbsd
 
 all:	lpr
--- lpr-0.72/lprm/Makefile.orig	Mon Mar 24 17:39:39 2003
+++ lpr-0.72/lprm/Makefile	Mon Mar 24 17:39:48 2003
@@ -1,6 +1,6 @@
 
-CFLAGS	= -Wall $(RPM_OPT_FLAGS) -I../common_source -I/usr/include/bsd
-LDFLAGS	= 
+CFLAGS	= -Wall $(OPT_FLAGS) -I../common_source -I/usr/include/bsd
+LDFLAGS	= -s
 LDLIBS = -lcommon -L ../common_source # -lbsd
 
 all:	lprm
--- lpr-0.72/lptest/Makefile.orig	Thu May 17 23:53:46 2001
+++ lpr-0.72/lptest/Makefile	Mon Mar 24 17:40:06 2003
@@ -1,6 +1,6 @@
 
-CFLAGS	= -Wall $(RPM_OPT_FLAGS) -I../common_source -I/usr/include/bsd
-LDFLAGS	= 
+CFLAGS	= -Wall $(OPT_FLAGS) -I../common_source -I/usr/include/bsd
+LDFLAGS	= -s
 LDLIBS = -lcommon -L ../common_source # -lbsd
 
 all:	lptest
--- lpr-0.72/pac/Makefile.orig	Mon Mar 24 17:40:31 2003
+++ lpr-0.72/pac/Makefile	Mon Mar 24 17:40:51 2003
@@ -1,6 +1,6 @@
 
-CFLAGS	= -Wall $(RPM_OPT_FLAGS) -I../common_source -I/usr/include/bsd
-LDFLAGS	= 
+CFLAGS	= -Wall $(OPT_FLAGS) -I../common_source -I/usr/include/bsd
+LDFLAGS	= -s
 LDLIBS = -lcommon -L ../common_source # -lbsd
 
 all:	pac
--- lpr-0.72/Makefile.orig	Mon Mar 24 17:47:38 2003
+++ lpr-0.72/Makefile	Mon Mar 24 17:49:18 2003
@@ -2,8 +2,9 @@
 
 VERSION=$(shell awk '/^Version:/ { print $$2 }' lpr.spec)
 
-CFLAGS	= -Wall $(RPM_OPT_FLAGS) -I../common_source -I/usr/include/bsd
-LDFLAGS	= 
+CFLAGS	= -Wall $(OPT_FLAGS) -I../common_source -I/usr/include/bsd
+LDFLAGS	= -s
+MANDIR = /usr/share/man
 LDLIBS = -lcommon -L ../common_source # -lbsd
 
 SUBDIRS	= common_source filters lpc lpd lpq lpr lprm lptest pac
