--- lpr-0.72/lpd/printjob.c.orig	2020-08-22 22:38:56.591103509 +0200
+++ lpr-0.72/lpd/printjob.c	2020-08-22 23:34:38.119667564 +0200
@@ -582,7 +582,7 @@
 	FILE *fp;
 	char *av[15], buf[BUFSIZ];
 	int pid, p[2], stopped = 0;
-	union wait status;
+	int status;
 	struct stat stb;
         char *optspace;
         int optlen;
@@ -751,12 +751,12 @@
 	fo = pfd;
 	if (ofilter > 0) {		/* stop output filter */
 		write(ofd, "\031\1", 2);
-		while ((pid=wait3((int *)&status, WUNTRACED, 0)) > 0 && pid != ofilter);
+		while ((pid=wait3(status, WUNTRACED, 0)) > 0 && pid != ofilter);
 		if (!WIFSTOPPED(status)) {
 			(void) close(fi);
 			syslog(LOG_WARNING,
 				"%s: output filter died (retcode=%d termsig=%d)",
-				printer, status.w_retcode, status.w_termsig);
+				printer, WEXITSTATUS(status), WTERMSIG(status));
 			return(REPRINT);
 		}
 		stopped++;
@@ -809,9 +809,9 @@
 	}
 	(void) close(fi);
 	if (child < 0)
-		status.w_retcode = 100;
+		status = W_EXITCODE(100, WTERMSIG(status));
 	else
-		while ((pid = wait((int *)&status)) > 0 && pid != child)
+		while ((pid = wait(status)) > 0 && pid != child)
 			;
 	child = 0;
 	prchild = 0;
@@ -832,10 +832,10 @@
 
 	if (!WIFEXITED(status)) {
 		syslog(LOG_WARNING, "%s: filter '%c' terminated (termsig=%d)",
-			printer, format, status.w_termsig);
+			printer, format, WTERMSIG(status));
 		return(ERROR);
 	}
-	switch (status.w_retcode) {
+	switch (WEXITSTATUS(status)) {
 	case 0:
 		tof = 1;
 		return(OK);
@@ -845,7 +845,7 @@
 		return(ERROR);
 	default:
 		syslog(LOG_WARNING, "%s: filter '%c' exited (retcode=%d)",
-			printer, format, status.w_retcode);
+			printer, format, WEXITSTATUS(status));
 		return(FILTERERR);
 	}
 }
